new-pipeline/evo.html

```diff
@@ -54,6 +54,13 @@ <h2 style="font-size: 18px; margin-bottom: 0">
                     as
                     <select id="filter-process-type" class="multiselect" title="Process type" data-all-selected="any process type"></select>
                 </h2>
+                <hr></hr>
+                <div class="caption-div">
+                <figcaption id="evo-caption" class="overflow ellipsis"></figcaption>
```

Comment:
Likewise, lets use `evo-caption`. This and the following element are missing indentation (should be 2 spaces per level).

Reply:
Here, you mean `evo-caption-text` Right?